apps/demo.namerank.io/app/results.tsx

```diff
@@ -0,0 +1,85 @@
+import { createClient } from "@namehash/namerank";
+import { Indicator } from "./indicator";
+
+const namerank = createClient({
+  namerankEndpoint:
+    "https://izzkysqb6d6qzhnpv4ybqyty2e0ktjwe.lambda-url.us-east-1.on.aws/namerank",
+});
+
+interface ResultsProps {
+  name: string;
+}
+
+export default async function Results({ name }: ResultsProps) {
+  const result = await namerank.inspectName(encodeURIComponent(name), {});
```

Comment:
It doesn't seem right to me that we should be doing an `encodeURIComponent` outside the call to `inspectName`. This should happen inside the SDK not outside of it.

Reply:
It was just belt and braces during my dev before I added the SDK. We can remove it, no big deal.